cpplint.py

```diff
@@ -518,7 +520,27 @@ def u(x):
     xrange = range
     itervalues = dict.values
     iteritems = dict.items
+=======
+Py3k = (sys.version_info[0] == 3)
+"""A boolean to check if we are running Python3000"""
+
+try:
+    xrange(0,1)
+except NameError:
+    xrange = range
+try:
+    unicode
+except NameError:
+    basestring = unicode = str
+try:
+    long
+except NameError:
+    long = int
+>>>>>>> Added support for Python3000
+
```

Comment:
All of the above code just introduces polyfills for Python 3. The code base is Python 2 so I just filled in the bits that Python 3 doesn't understand.

 * `basestring = unicode = str` fills in the gaps on Python 3. A `str` _is_ `unicode` on Python 3.
 * `xrange = range` is correct, `range` in Python 3 does exactly what `xrange` did in Python 2
 * Redefining `iteritems` and `itervalues` is fine
 * `long = int` is correct. `int`s are `long`s in Python 3

TBH, I don't really mind what the compatibility changes are, as long as the linter runs on both!